frontend/web/components/pages/EnvironmentSettingsPage.js

```diff
@@ -224,15 +223,18 @@ const EnvironmentSettingsPage = class extends Component {
     )
   }
 
-  confirmToggle = (title, description, feature) => {
+  confirmToggle = (title, environmentProperty, environmentPropertyValue) => {
     openModal(
       title,
       <ConfirmToggleEnvFeature
-        description={`${description} Are you sure that you want to change this value?`}
-        feature={feature}
-        featureValue={this.state[feature]}
-        onToggleChange={(value) => {
-          this.setState({ [feature]: value }, this.saveEnv)
+        description={'Are you sure that you want to change this value?'}
```

Comment:
This should ideally explain what the change will do

Reply:
This is how it looks now:

<img width="508" alt="Screenshot 2024-06-20 at 9 20 52 AM" src="https://github.com/Flagsmith/flagsmith/assets/41410593/2c415d52-2906-4bea-a410-1d981e46c1e5">


VS Before:

<img width="508" alt="Screenshot 2024-06-20 at 9 27 45 AM" src="https://github.com/Flagsmith/flagsmith/assets/41410593/ee5cfb34-21c6-49d1-b625-212a55fbabec">

Reply:
Should be in human readable form not snake case.. also what does this do?

Reply:
<img width="599" alt="Screenshot 2024-06-25 at 11 56 16 AM" src="https://github.com/Flagsmith/flagsmith/assets/41410593/7cd0580a-fa3c-4b44-9a40-56ee3333da10">
@kyle-ssg  This is the new version